src/patternfly/components/Dropdown/docs/code.md

```diff
@@ -40,3 +41,4 @@ The dropdown menu can contain either links or buttons, depending on the expected
 | `.pf-m-plain` | `.pf-c-dropdown__toggle` | Modifies to display the toggle with no border |
 | `.pf-m-active` | `.pf-c-dropdown__toggle` | Forces display of the active state of the element. This state is primarily for demonstration purposes and would not normally be used in lieu of the `:active` pseudo-class. |
 | `.pf-m-disabled` | `a.pf-c-dropdown__menu-item` | Modifies to display the menu item as disabled. This applies to `a.pf-c-dropdown__menu-item` and should not be used in lieu of the `disabled` attribute on `button.pf-c-dropdown__menu-item`|
+| `.pf-m-disabled` | `div.pf-c-dropdown__toggle` | Modifies to display the dropdown toggle as disabled. This applies to `div.pf-c-dropdown__toggle` and should not be used in lieu of the `disabled` attribute on `button.pf-c-dropdown__toggle`. When this is used, `disabled` should also be added to any form elements in `div.pf-c-dropdown__toggle`|
```

Comment:
I like this description. 👍

Reply:
:D @jgiardino not sure if you had a chance to review the rest of the PR or not, but when you can would you mind leaving a review?